notebooks/swarm.ipynb

```diff
@@ -0,0 +1,1106 @@
+{
```

Comment:
<p>* typo in "paramters"</p><p>* Also, we can say that parameters are "a name, a generator and a system message with instructions"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>